micro-rdk/build.rs

```diff
@@ -2,6 +2,12 @@ use regex::Regex;
 
 fn main() {
     println!("cargo::rustc-check-cfg=cfg(esp32)");
+    println!("cargo::rustc-check-cfg=cfg(esp32s2)");
```

Comment:
[q] - Does this belong inside the `if Regex...` block below?

Reply:
probably

Reply:
Moved it